### DISCUSSION TITLE
Not working!

### DISCUSSION THREAD

**Discussion by maximegti** [rustdesk/rustdesk#7214] [General]
Rustdesk servers have not been reachable for many weeks. Is there hope that the problem will be solved?

**Comment by rustdesk** [comment]
https://twitter.com/rustdesk/status/1754818959637942671

**Reply by maximegti** [reply]
"we black list a lot of suspicious ip". I try it from many IP: from my office and it doesn't works; then I tried from my Home, from my mom's home, from my sister's home and also in hotspot from my smartphone. It doesn't even work from any of these places. Is it normal? It doesn't seem related to the public IP I was connected from...
